schema/me/sale_registrations.js

```diff
@@ -0,0 +1,45 @@
+import { first } from 'lodash';
+import gravity from '../../lib/loaders/gravity';
+import Sale from '../sale';
+import Sales from '../sales';
+import Bidder from '../bidder';
+import {
+  GraphQLList,
+  GraphQLBoolean,
+  GraphQLObjectType,
+} from 'graphql';
+
+export const SaleRegistrationType = new GraphQLObjectType({
+  name: 'SaleRegistration',
+  fields: () => ({
+    is_registered: {
+      type: GraphQLBoolean,
+    },
+    bidder: {
+      type: Bidder.type,
+    },
+    sale: {
+      type: Sale.type,
+    },
+  }),
+});
+
+export default {
+  type: new GraphQLList(SaleRegistrationType),
+  args: Sales.args,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
+    return gravity('sales', options)
```

Comment:
Ah ha! I had wondered if the args were being leveraged like that here. In that case :+1: